Zilon.Core/Zilon.Core.Specs/Contexts/FeatureContextBase.cs

```diff
@@ -124,22 +124,31 @@ public async Task CreateSectorAsync(int mapSize)
             var sectorManager = ServiceProvider.GetRequiredService<ISectorManager>();
             var humanPlayer = ServiceProvider.GetRequiredService<HumanPlayer>();
 
-            var locationScheme = new TestLocationScheme
+            var sectorSubScheme = new TestSectorSubScheme
             {
-                SectorLevels = new ISectorSubScheme[]
-                {
-                    new TestSectorSubScheme
-                    {
-                        RegularMonsterSids = new[] { "rat" },
-                    }
+                RegularMonsterSids = new[] { "rat" },
+                MapGeneratorOptions = new SquareGenerationOptionsSubScheme
+                { 
+                    Size = mapSize
                 }
             };
-            var globeNode = new GlobeRegionNode(0, 0, locationScheme);
-            humanPlayer.GlobeNode = globeNode;
+
+            var sectorNodeMock = new Mock<ISectorNode>();
+            sectorNodeMock.SetupGet(x=>x.State).Returns(SectorNodeState.SectorMaterialized);
+            sectorNodeMock.SetupGet(x => x.SectorScheme).Returns(sectorSubScheme);
+            var sectorNode = sectorNodeMock.Object;
+
+            humanPlayer.BindSectorNode(sectorNode);
 
             await sectorManager.CreateSectorAsync();
         }
 
+        private class SquareGenerationOptionsSubScheme : ISectorSquareMapFactoryOptionsSubScheme
+        {
+            public SchemeSectorMapGenerator MapGenerator { get => SchemeSectorMapGenerator.SquarePlane; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'MapGenerator'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)